### PR TITLE
Feature/wire up settings page

### PR DESCRIPTION
**Pull request by renburnett**
#### Please check if the PR fulfills these requirements

- [x] The changes don't come from your master branch. ([Source](https://blog.jasonmeridth.com/posts/do-not-issue-pull-requests-from-your-master-branch/)) 

---

#### What kind of change does this PR introduce? (bug fix, feature, docs update, ...)

- [x] New feature (non-breaking change which adds functionality)

#### What is the current behavior? (You can also link to an open issue here)

settings page pulls static info

#### What is the new behavior? (if this is a feature change)

settings page pulls dynamic data from backend

#### Does this PR introduce a breaking change? (What changes might users need to make in their application due to this PR?)

not that i know of

### CODE REVIEW

File: src/Components/Button/index.js
Comment (tackc) [inline src/Components/Button/index.js:1]: Good call adding the defaultProps

File: src/Components/Button/style.module.scss
Comment (tackc) [inline src/Components/Button/style.module.scss:1]: So much cleaner...good call making this a sass instead!

File: src/Components/Icon/index.js
Comment (tackc) [inline src/Components/Icon/index.js:1]: Absolutely! Good call. Having preset size options will make implementation much more consistent!

File: src/Pages/SettingsPage/style.module.scss [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+}
+
+.passwordContainer {
+  margin-top: 2em;
+  display: flex;
+  justify-content: space-between;
+}
+
+.editLink {
+  font-family: Open Sans;
```
Comment (tackc) [inline src/Pages/SettingsPage/style.module.scss:113]: I'm pretty sure this just came over from the previous css file, but does Open Sans need to be in quotes? Also, it seems like we have the `font-family` sprinkled across a ton of stylesheets. Can we just set this globally so it's not necessary to set everywhere individually?

File: src/Pages/SettingsPage/style.module.scss [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+  font-style: normal;
+  font-weight: bold;
+  color: var(--NAVY_BLUE);
+  text-decoration: none;
+  margin-top: 0.6em;
+}
+
+.infoContainer {
+  margin-top: 1em;
+  font-family: Open Sans;
```
Comment (tackc) [inline src/Pages/SettingsPage/style.module.scss:123]: double quotes?

**Review by tackc — APPROVED** [review]
The main "issue" I noticed was regarding the `font-family: Open Sans;`. That said, the issue already existed. We should probably set the font-family globally to alleviate sprinkling it in throughout various stylesheets. That can absolutely happen in a separate PR

File: src/Pages/SettingsPage/style.module.scss (hunk 1 above)
Comment (renburnett) [reply src/Pages/SettingsPage/style.module.scss:113]: @tackc makes total sense 😆 I also noticed that each page has a `container` wrapper that we could prob move to a single parent

File: src/Pages/SettingsPage/style.module.scss (hunk 2 above)
Comment (renburnett) [reply src/Pages/SettingsPage/style.module.scss:123]: @tackc i'll add this to a trello card something like "remove redundant `font-family`s everywhere"

**Pull request merged**